client/signup/config/steps-pure.js

```diff
@@ -359,7 +349,7 @@ export function generateSteps( {
 		/* Imports */
 		'from-url': {
 			stepName: 'from-url',
-			providesDependencies: [ 'importUrl', 'themeSlugWithRepo' ],
+			providesDependencies: [ 'importSiteDetails', 'importUrl', 'themeSlugWithRepo' ],
```

Comment:
Are there other site details that are significant other than the engine? From a glance through this PR it doesn't seem so, but I may be missing something.

Reply:
The source URL -- the only other place it lives in state is for the controlled input on the signup side.  I don't mind splitting everything out, but this works :)

Reply:
Nice cheers, must have just missed it :)